releasenotes/notes/old-agent-8193550a76b21357.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    tracer: support for Datadog Agent v5 has been dropped.
```

Comment:
should we link to the support page for this?

Reply:
What would we link to?

Reply:
https://ddtrace.readthedocs.io/en/stable/versioning.html#release-support

I worry if someone doesn't realize that this isn't meant to be supported, so we are free to remove it

Reply:
:+1: great idea